releasenotes/notes/feat-submit-langchain-tool-spans-e7f76291dfa1efe5.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Langchain : Adds support for tracing tool invocations.
```

```suggestion
    langchain : Adds support for tracing tool invocations.
```
For integrations, we don't capitalize the library name in release notes. (not sure why, just standard practice)